### ISSUE TITLE
Please add DUB support

### ISSUE THREAD

**Issue by denizzzka** [Jebbs/DSFMLC#11]

**Comment by aubade** [comment]
This is not currently possible; Dub can only compile D code, not C++.

**Comment by Jebbs** [comment]
Correct. Even if we got rid of the back end, almost nothing would change as far as dub is concerned.
